.github/workflows/breaking-change-review.yml

```diff
@@ -0,0 +1,90 @@
+name: PR Review Workflow for Breaking Changes
+
+on:
+  pull_request_review:
```

Comment:
Just realised that we probably still need `pull_request: [opened]` here, otherwise the check won't appear until some review appear.

Reply:
Also, `pull_request: [labeled, unlabeled]` will also be needed to track `breaking change` add/removal.